WordPress/twentytwentyfive
# Add typography block style variations

**Description**
This is an experiment for https://github.com/WordPress/twentytwentyfive/issues/305

This does not replace the custom font sizes in theme.json, it is a complement that helps us remove hard coded font sizes in the markup.
This means that the size and line height can be adjusted in a global style variation, _if we want._

What was changed:
- A new block style variation for the paragraph block was added, with font size XL and line height 1.2. The font style was then applied to a section of a block pattern, replacing the size and line height in the markup.
- A new block style variation for the paragraph, heading and post-terms block was added, with font size small, uppercase, font-weight 700, and letter-spacing 1.4px.


**Screenshots**
This screenshot shows the "Lead" paragraph style variation, text with the old markup first, and the new variation second: There should be no visible difference:
<img width="924" alt="p-xl" src="https://github.com/user-attachments/assets/e8a0e550-80ea-4dc9-9fda-d646debbfb86">

This video shows how the style is applied:
https://github.com/user-attachments/assets/af5eb091-3efa-412c-9c91-b8b5b1cbc8be

**Testing Instructions**

Place a paragraph, a post terms block and two heading blocks: One of the headings should use the H6 level.
Apply the different style variations, and compare the result in the editor and front.
The uppercase variation should match the style of the H6.
